pymeasure/instruments/wentworthlabs/s200.py

```diff
@@ -0,0 +1,548 @@
+        get_process=lambda r:
+        float(r.replace("RZIM ", ""))
+    )
+
+    z_grosslift = Instrument.control(
+        "RKGM", "WKGM %d",
+        "Control the z-axis gross lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_GROSSLIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKGM ", ""))
+    )
+
+    z_finelift = Instrument.control(
+        "RKFM", "WKFM %d",
+        "Control the z-axis fine lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_FINELIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKFM ", ""))
+    )
+
+    indexing_mode = Instrument.setting(
+        "%s",
+        "Control the indexing mode."
+        "If True, Enters indexing mode and moves to the first die to be probed, or moves to the"
+        " next die to be probed if already in indexing mode. Indexing mode is exiting"
+        " using the NXF command (above). Pay attention. When in indexing mode, "
+        " next_die property will be not recognized as a command and will get the state of the table"
+        " out of communication."
+        " All subsequent commands will be ignored and thus we will re-enter in remote mode.",
+        # AWP compatible: Yes
+        validator=strict_discrete_set,
+        values={True: 'NXT', False: 'NXF'},
+        map_values=True,
+        check_set_errors=True
+    )
+
+    next_die = Instrument.setting(
+        "NX%s",
+        "Control the movement of the chuck to the next die",
+        # If probe_table.next_die = "D" then it will go below the current die
+        # If probe_table.next_die = "U" then it will go above the current die
+        # If probe_table.next_die = "L" then it will go to the left of the current die
+        # If probe_table.next_die = "R" then it will go to the right of the current die
+        # AWP compatible: Yes
+        check_set_errors=True,
+        validator=strict_discrete_set,
+        values=['D', 'U', 'L', 'R']
+    )
+
+    status_byte = Instrument.measurement(
+        "STA",
+        "Measure the status byte of the instrument",
+        get_process=lambda r:
+        r
+    )
+
+    extended_status_info = Instrument.measurement(
+        "STP",
+        "Measure the extended status information of the instrument",
+        get_process=lambda r:
+        r
+    )
+
+    exit_remote = Instrument.setting(
+        "ESC",
+        "Set the proble table in local mode. Exits remote mode."
+        # This command does not send a reply. However, when remote mode is re-entered,
+        # the Pegasus unit sends an INF_000 message
+    )
+
+    serial_number = Instrument.measurement(
+        "GSN",
+        "Measure the serial number of the probe table",
+        # AWP compatible: No
+        get_process=lambda r:
```

Comment:
Instead of `get_process=lambda r: str(r)`, I think using `cast=str` should work? [The `CommonBase.values` method hits earlier](https://github.com/pymeasure/pymeasure/blob/3c288c7909acc5cd3c36af425f41b1e891152bae/pymeasure/instruments/common_base.py#L472)  then `get_process`, and it takes in the `cast` argument.

Reply:
Exactly

Reply:
Hi. Thanks.